docs/DeveloperGuide.adoc

```diff
@@ -767,16 +825,74 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+=== Use case: Merge duplicate contacts
```

Comment:
Think current addressbook does not allow duplicate contacts, so you have to be a bit more specific here. E.g. merge contact with similar field?

Reply:
@tby1994 I wrote the use case based on your user story so let me know how you'd like to edit this.